coralproject/talk
# Plugin Fixes

## What does this PR do?

Fixes some issues related to plugin creation:

- Fixes #1690 

## How do I test this PR?

1. Remove your `plugins.json` via `mv plugins{,-old}.json`
2. Create a new plugin with `./bin/cli plugins create` and answer `Y` to adding it to the `plugins.json`.
3. Notice that the `plugins.json` file was created, and was based off of the `plugins.default.json`, and now contains your new plugin!
